css/style.css

```diff
@@ -105,7 +100,12 @@ header h2 {
 header ul {
   float: left;
   padding-top: 12px;
-  padding-left: 125px;
+  padding-left: 100px;
+}
+
+#findapodcast {
```

Comment:
use snake casing by using a "-" between each word. this makes it easier to read and is an understood practice for css selectors